Lombiq.Tests.UI/Extensions/HttpClientUITestContextExtensions.cs

```diff
@@ -24,7 +24,7 @@ namespace Lombiq.Tests.UI.Extensions;
         Justification = "Disposed by the HttpClient.")]
 public static class HttpClientUITestContextExtensions
 {
-    private static readonly JsonSerializerOptions JsonSerializerOptions = new(JsonSerializerDefaults.Web);
+    public static JsonSerializerOptions JsonSerializerOptions { get; set; } = new(JsonSerializerDefaults.Web);
```

Comment:
Why do you need a setter as well? I don't see the setter used anywhere, and it's generally not a safe thing for a static prop.

```suggestion
    public static JsonSerializerOptions JsonSerializerOptions { get; } = new(JsonSerializerDefaults.Web);
```